index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Asha</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <div class="toolbar">
+        <h3 class="logo">My logo</h3>
+        <i class="fas fa-bars"></i>
+    </div>
+        <div class="Ellipse11"><img src="\Assets\Ellipse 11.png"></div>
+        <div class="Ellipse3"><img src="\Assets\Ellipse 3.png"></div>
+        <div class="Rectangle4"><img src="\Assets\Rectangle 4.png"></div>
+        <div class="Rectangle7"><img src="\Assets\Rectangle 7.png"></div>
+        <div class="Ellipse5"><img src="\Assets\Ellipse 5.png"></div>
+        <div class="Ellipse8"><img src="\Assets\Ellipse 8.png"></div>
```

Comment:
Kindly remove these elements, decorations are always background images, not img tags. Please use CSS to add this as background. :recycle: